### ISSUE TITLE
Support mounting volume on Fargate ecs param

### ISSUE THREAD

**Issue by Mehul1313** [aws/amazon-ecs-cli#807]
### Summary
Support mounting volume on Fargate ecs param. Does this support exist already? I am finding that it is only supported for EC2 launch type.

**Comment by SoManyHs** [comment]
Hi @Mehul1313,

Are you referring to ephemeral task storage? You should be able to specify it in the container definition. Could you include what your `docker-compose.yml` and `ecs-params.yml` files look like, as well as which ECS CLI command you are trying to run?

**Comment by Mehul1313** [comment]
Yes, I basically want to mount the non-persistent volume to the container deployed through ecs-cli as explained in the https://docs.aws.amazon.com/AmazonECS/latest/developerguide/fargate-task-storage.html

This is docker-compose.yaml look like
..
10:59:20      logging:
10:59:20        driver: awslogs
10:59:20        options:
10:59:20          awslogs-group: mlp-sandbox
10:59:20          awslogs-region: us-east-1
10:59:20          awslogs-stream-prefix: my-service
10:59:20      volumes:
10:59:20      - data-volume:/opt/foo
10:59:20  version: '3'
10:59:20  volumes:
10:59:20    foo: null

The issue is that it only get 4GB storage mounted at /opt/foo. Can this limit be changed?

I have not come across any AWS documentation on what type of docker volumes are supported. 
I use ecs-cli compose service up and ecs-cli service scale  up command to bring the service up. 

Any help is appreciated. thanks
